package.json

```diff
@@ -55,6 +56,8 @@
     "postcss": "^5.2.0",
     "postcss-discard-duplicates": "^2.0.2",
     "postcss-loader": "^1.1.0",
+    "pouchdb": "^6.1.2",
+    "pouchdb-find": "^0.10.5",
```

Comment:
At first `yarn install`, yarn was complaining about missing dependency `pouchdb`. It seems to be ok now. 👍